libreoffice-core/include/vcl/outdev.hxx

```diff
@@ -1156,6 +1156,7 @@ public:
                                                    const OUString& rReplaceFontName,
                                                    AddFontSubstituteFlags nFlags );
     static void                 RemoveFontsSubstitute();
+    static void                 AddCustomMacroFonts();
```

Comment:
```suggestion
    // MACRO-1518: Fix Calibri and Cambria display
    static void                 AddCustomMacroFonts();
```

For modifications in existing LOK C++ headers and code, we've made it a practice to add a comment with the `MACRO-#` and a brief description of the problem it's intended to solve.

For single lines, adding a comment above the line. For multiple lines, adding an identical comment above and below.

This makes it easier to resolve conflicts when pulling changes from upstream and easily identify the problem that it was intended to resolve, in case upstream resolves it for us later on.

Reply:
Added comments around changes